infra/src/main/java/com/redhat/service/bridge/infra/exceptions/ErrorInMemoryDAO.java

```diff
@@ -0,0 +1,116 @@
+package com.redhat.service.bridge.infra.exceptions;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.redhat.service.bridge.infra.models.ListResult;
+import com.redhat.service.bridge.infra.models.QueryInfo;
+import com.redhat.service.bridge.infra.utils.Constants;
+
+import io.quarkus.runtime.Quarkus;
+
+@ApplicationScoped
+public class ErrorInMemoryDAO implements ErrorDAO {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ErrorInMemoryDAO.class);
+    private static final ObjectMapper MAPPER = new ObjectMapper();
```

Comment:
I know this is picky, but MAPPER and LOGGER are not literal constants, but final references, and I do not think they should be in uppercase 
see https://stackoverflow.com/a/12069687/16843807

Reply:
Well, if you really want to discuss such things https://rules.sonarsource.com/java/tag/convention/RSPEC-1312 ;) The brutal truth is that there is no rule on this regard, only conventions. If you really have a strong opinion on this, I'll make them lowercase

Reply:
I think we need a convention for singleton instance naming (which are private final references, but different from literal constants, than since the 70s has been in uppercase in all programming languages) , but it is outside the scope of this JIRA. 
Im not stronger support of conventions, because
- I do not like to put { when there is just one statement after an if, but there is a convention and I have to add an uneeded bracket  because of conventions
- My whole programmer life I haves used  { in new line, but there is a google convention to put it on the same line. 
and so on. 
Lets discuss on a different forum